drivers/scheduler/rke/rke.go

```diff
@@ -97,17 +97,20 @@ func (r *Rancher) UpdateRancherClient(clusterName string) error {
 	var rkeParametersValue RancherClusterParameters
 	var err error
 	var rkeToken string
-	masterNodeName := node.GetMasterNodes()[0].Name
-	endpoint := "https://" + masterNodeName + "/v3"
+	var endpoint string
 	if clusterName == "destination-config" {
 		rkeToken = os.Getenv("DESTINATION_RKE_TOKEN")
-		if rkeToken == "" {
-			return fmt.Errorf("env variable DESTINATION_RKE_TOKEN should not be empty")
+		endpoint = os.Getenv("DESTINATION_RKE_URL")
+		log.Infof("Destination endpoint: %v", endpoint)
+		if rkeToken == "" || endpoint == "" {
+			return fmt.Errorf("env variable DESTINATION_RKE_TOKEN or DESTINATION_RKE_URL should not be empty")
 		}
 	} else if clusterName == "source-config" {
 		rkeToken = os.Getenv("SOURCE_RKE_TOKEN")
-		if rkeToken == "" {
-			return fmt.Errorf("env variable SOURCE_RKE_TOKEN should not be empty")
+		endpoint = os.Getenv("SOURCE_RKE_URL")
+		log.Infof("Source endpoint: %v", endpoint)
+		if rkeToken == "" || endpoint == "" {
+			return fmt.Errorf("env variable SOURCE_RKE_TOKEN or SOURCE_RKE_URL should not be empty")
```

Comment:
@snigdha-px Do we really need the check here. I believe we can create only one Rancher cluster and add both source & destination clusters to same Rancher cluster. In that case we no need to export two env variables and no check required.
@sudas-px Need your inputs here.